test/WebSites/BasicWebSite/Areas/Area1/Controllers/RemoteAttribute_HomeController.cs

```diff
@@ -19,23 +19,23 @@ public IActionResult Create()
         }
 
         [HttpPost]
-        public IActionResult Create(Person person)
+        public IActionResult Create(RemoteAttributeUser user)
         {
             ModelState.Remove("id");
             if (!ModelState.IsValid)
             {
-                return View(person);
+                return View(user);
             }
 
-            _person = person;
+            _user = user;
             return RedirectToAction(nameof(Details));
         }
 
-        [Route("/[Area]", Name = "AriaHome", Order = -3)]
-        [Route("/[Area]/[Controller]/Index", Order = -2)]
+        [HttpGet]
+        [Route("/[area]/[controller]/[action]", Name = "Area1Home")]
```

Comment:
Why does this action now have `[HttpGet]`? Why have the URLs which will reach this action changed?